runRAPL.py

```diff
@@ -176,6 +242,10 @@ def handle_results(res, raw_results_csv, stats):
 
     benchmark_programs = get_benchmark_programs(benchmarks, paradigms, languages)
 
-    perform_benchmarks(benchmark_programs, iterations, time_limit, output_file, skip_build, skip_runs)
+    if args.sestoft_approach:
+        perform_benchmarks(benchmark_programs, iterations, time_limit, output_file, skip_build, skip_runs)
+    else:
+        perform_benchmarks_statistical(benchmark_programs, output_file)
```

Comment:
Ville nok tage hver af dem her ud i sin egen fil, for overskuelighedens skyld. Og så kalde perform_benchmarks noget med Sestoft